webapp/graphite/local_settings.py.example

```diff
@@ -65,13 +65,15 @@
 #                        (21600, 180)] # >= 6 hour queries are cached 3 minutes
 #MEMCACHE_KEY_PREFIX = 'graphite'
 
-
 # This lists the memcached options. Default is an empty dict.
 # Accepted options depend on the Memcached implementation and the Django version. 
 # Until Django 1.10, options are used only for pylibmc. 
 # Starting from 1.11, options are used for both python-memcached and pylibmc.
 #MEMCACHE_OPTIONS = { 'socket_timeout': 0.5 }
 
+# this setting controls the default xFilesFactor used for query-time aggregration
```

Comment:
this could be a bit clearer.  AFAIK "query-time aggregation" is new terminology that we haven't used before.
exactly which functionality does this apply to? I see 3 cases:
A) runtime consolidation (over time. as driven by consolidateBy)
B) math functions over time (like movingAverage)
C) math functions that work across series but not over time (like sumSeries, avg); 

to which does this apply?

Reply:
All of them

Reply:
If you want more fine-grained control over the xff for individual cases you can use the new function to set it exactly as needed

Reply:
that's great but i do think we should clean up the terminology to be more clear.  my suggestion is to instead of introducing the new terminology "query-time aggregation" , say "math functions and runtime consolidation" or phrase it like it's phrased in the docs for the setXFilesFactor function.

Reply:
I think it's fine as is, as you note the docs themselves are pretty clear about what it affects.  If it bothers you I'd be happy to accept a PR.